### PR TITLE
Add more move tests

### PR DESCRIPTION
**Pull request by orestisfl**

### CODE REVIEW

File: test/test_move.py [hunk 1]
```diff
@@ -165,3 +165,103 @@ def test_move_changes_workspace_layout(sway):
...
+    -> move right
+    -> move left
+    ((container L_VERT (view2) (view3)) (view1 focus) (view4))
+    '''
+    view1 = sway.open_window('view1')
+    view1.command('splitv')
+    view2 = sway.open_window('view2')
+    view3 = sway.open_window('view3')
+    view4 = sway.open_window('view4')
+    sway.ipc.command('move right')
```
Comment (bpinto) [inline test/test_move.py]: Is this necessary? I don't see this on the test description.

Comment (orestisf1993) [reply test/test_move.py]: The description is meant to say: 1 horizontal container with one vertical container with 3 windows and one window.

Comment (bpinto) [reply test/test_move.py]: :+1: Missed the `()`.

**Comment by acrisci** [comment]
Actually I think this is ok. I need to work out a consistent style for tests still though.

**Pull request merged**